Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/helper/ImagePickDelegate.java

```diff
@@ -87,33 +96,66 @@ public void pick(ImagePickCallback callback, boolean longPressed) {
                 try {
                     clipboardURL =
                             HttpUrl.get(getClipboardManager().getPrimaryClip().getItemAt(0).getText().toString());
-                } catch (Exception ignored) {
-                    showToast(R.string.image_url_get_failed);
+                } catch (Exception exception) {
+                    String message = getAppContext().getString(
```

Comment:
You've got this sort of method call everywhere, instead of just using getString directly from AndroidUtils. I added those utility methods for a reason!

Reply:
Yeah, just a habit.

Reply:
Done.